app/assets/javascripts/index.js

```diff
@@ -0,0 +1,235 @@
+// ESLINT:
+/* global L, moment, $ */
+
+
+// Load all the channels within this directory and all subdirectories.
+// Channel files must be named *_channel.js.
+
+// Const channels = require.context('.', true, /_channel\.js$/)
+// Channels.keys().forEach(channels)
+
+'use strict';
+
+window.addEventListener('load', () => {
+	const greenIcon = new L.Icon({
+		iconUrl: 'https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-green.png',
+		shadowUrl: 'https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/images/marker-shadow.png',
+		iconSize: [25, 41],
+		iconAnchor: [12, 41],
+		popupAnchor: [1, -34],
+		shadowSize: [41, 41]
+	});
+	const blueIcon = new L.Icon({
+		iconUrl: 'https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-blue.png',
+		shadowUrl: 'https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/images/marker-shadow.png',
+		iconSize: [25, 41],
+		iconAnchor: [12, 41],
+		popupAnchor: [1, -34],
+		shadowSize: [41, 41]
+	});
+	const yellowIcon = new L.Icon({
+		iconUrl: 'https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-yellow.png',
+		shadowUrl: 'https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/images/marker-shadow.png',
+		iconSize: [25, 41],
+		iconAnchor: [12, 41],
+		popupAnchor: [1, -34],
+		shadowSize: [41, 41]
+	});
+
+	let birdData = null;
+	let birdImages = null;
+	const mymap = L.map('mapid').setView([37.42, -121.91], 13);
+	L.tileLayer('https://a.tiles.mapbox.com/styles/v1/lohneswright/ciocejooj006obdnjhmd2x9qp/tiles/{z}/{x}/{y}?access_token=pk.eyJ1IjoibG9obmVzd3JpZ2h0IiwiYSI6IngzbVlqNnMifQ.OwxjrBKoGXc60NB5x6GKzw', {
+		maxZoom: 22
+	}).addTo(mymap);
+
+	const legend = L.control({
+		position: 'topright'
+	});
+	legend.onAdd = function(map) {
+		const div = L.DomUtil.create('div', 'legend');
+		div.innerHTML += '<img src="https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-2x-green.png" height=41 width=25>Your Location<br>';

```

Comment:
Suggest adding alt="Green marker", alt="Gold marker", alt="Blue marker" to each of these image tags.